sources/web/datalab/polymer/modules/file-manager/file-manager.ts

```diff
@@ -139,12 +132,6 @@ interface FileManager {
   // one call.
 
   /**
-   * Returns true if we can host notebooks on this filesystem, and thus should
-   * display the Status column in the file browser.
-   */
-  canHostNotebooks(): boolean;
```

Comment:
Also should be removed from bigquery-file-manager.

Reply:
Done.